CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Fixed
+
+## 0.1.0 - 2023-06-16
```

Comment:
Did we decided not to provide `_VERSION` for cartridge roles?

Reply:
It is already implemented:

https://github.com/tarantool/sharded-queue/blob/7c98cc1b075122e09f597eda5291bb910e1dc7d0/sharded_queue/api.lua#L686

https://github.com/tarantool/sharded-queue/blob/7c98cc1b075122e09f597eda5291bb910e1dc7d0/sharded_queue/storage.lua#L125

Thank you for the notice, I've bumped the version.